test/sql/attach/attach_filepath_roundtrip.test

```diff
@@ -0,0 +1,96 @@
+# name: test/sql/attach/attach_filepath_roundtrip.test
+# description: Test file path roundtripping and concurrency
+# group: [attach]
+
+require noforcestorage
+
+statement ok
+PRAGMA enable_verification
```

Comment:
Could you add a test with `concurrentloop` similar to `test/sql/catalog/dependencies/test_concurrent_alter.test`